users/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+class User(models.Model):
+    user_id	     = models.CharField(max_length=40)
```

Comment:
```suggestion
class User(models.Model):
    user_name	     = models.CharField(max_length=40, unique=True)
```
- 회원가입 할 때 사용하는 ID를 위한 필드 같은데, 보통 user_name으로 사용합니다. 그리고 Id 같은 경우 유일한 값이여야 하기 때문에 unique 속성을 걸어주면 좋습니다. user_name 이외에도 unique 속성을 필요한 곳에 추가해 주세요.
- = 기준으로 정렬 잘 해주세요.